kqueen/server.py

```diff
@@ -10,11 +10,14 @@
 from flask import Flask
 from flask_jwt import JWT
 from flask_swagger_ui import get_swaggerui_blueprint
+from kqueen.utils.loggers import setup_logging
 from werkzeug.contrib.cache import SimpleCache
 
 import logging
 
-logger = logging.getLogger(__name__)
+# Logging configuration
+setup_logging('/code/kqueen/utils/logger_config.yml')
```

Comment:
How is this path going to be created in on machine? This will work only in container.

Reply:
yep, i already started to wrap it on  kqueen config-vars